src/lib/backend.ai-client-esm.ts

```diff
@@ -4060,6 +4061,7 @@ class Group {
    * };
    */
   async list(
+    type = ["GENERAL", "MODEL_STORE"],
```

Comment:
For better compatibility, it's better to add a new argument as the last argument.